README.md

```diff
@@ -37,16 +123,29 @@ Execute the following gradle task to start the go-server
 
 You can now access the [go-server via port 8153](http://localhost:8153)
 
-#### reload
+##### reload
 
 If you like to reload the go-server with new build run,
 ```bash
 ./gradlew clean restartGoCd
 ```
 
-#### stop
+##### stop
 
 You can stop the running docker instance with the following gradle task
 ```bash
 ./gradlew clean stopGoCd
 ```
+
+#### Releasing
+
+Released via the [gocd-plugin-gradle-task-helpers](https://github.com/gocd/gocd-plugin-gradle-task-helpers) onto
+GitHub.
+
+* Check the`gocdPlugin.pluginVersion` version in `build.gradle` and bump if necessary
+* Tag and publish
+    ```bash
+    ./gradlew githubRelease                       
+    PRERELEASE=no ./gradlew githubRelease
+    ```
+* Edit the release description on `GitHub` if necessary.
```

Comment:
Can we add a section for Upgrade guide?

The plugin ID got changed from 'GitPathMaterial' to 'git-path', it's worth mentioning it for existing user to use this plugin upgrade to the newer version without much tussle

Reply:
Good idea; will do that tonight.

Reply:
I have added this now.